docs-source/content/samples/simple/azure-kubernetes-service/model-in-image.md

```diff
@@ -0,0 +1,836 @@
+
+```bash
+# cd kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/model-in-image
+$ kubectl apply -f helm-sa-cluster-admin-role.yaml
+clusterrolebinding.rbac.authorization.k8s.io/helm-user-cluster-admin-role created
+```
+
+Create a namespace and service account for the operator.
+
+```bash
+$ kubectl create namespace sample-weblogic-operator-ns
+namespace/sample-weblogic-operator-ns created
+
+
+$ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
+serviceaccount/sample-weblogic-operator-sa created
+```
+
+Validate the service account was created with this command.
+
+```bash
+$ kubectl -n sample-weblogic-operator-ns get serviceaccount
+
+NAME                          SECRETS   AGE
+default                       1         9m24s
+sample-weblogic-operator-sa   1         9m5s
+```
+
+Install the operator. Ensure your current directory is `weblogic-kubernetes-operator`. It may take you several minutes to install the operator.
+
+```bash
+# cd weblogic-kubernetes-operator
+$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+  --namespace sample-weblogic-operator-ns \
+  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:3.1.1 \
+  --set serviceAccount=sample-weblogic-operator-sa \
+  --set "enableClusterRoleBinding=true" \
+  --set "domainNamespaceSelectionStrategy=LabelSelector" \
+  --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \
+  --wait
+
+NAME: weblogic-operator
+LAST DEPLOYED: Tue Nov 17 09:33:58 2020
+NAMESPACE: sample-weblogic-operator-ns
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Verify the operator with the following commands; the status will be `Running`.
+
+```bash
+$ helm list -A
+NAME                        NAMESPACE                     REVISION   UPDATED                                 STATUS       CHART                   APP VERSION
+sample-weblogic-operator    sample-weblogic-operator-ns   1          2020-11-17 09:33:58.584239273 -0700 PDT deployed     weblogic-operator-3.1
+
+
+$ kubectl get pods -n sample-weblogic-operator-ns
+NAME                                 READY   STATUS    RESTARTS   AGE
+weblogic-operator-775b668c8f-nwwnn   1/1     Running   0          32s
+```
+
+{{% notice note %}}
+You can sepcify the operator image by changing value of `--set image`. If you run into failures, see [Troubleshooting - WebLogic Kubernetes Operator installation failure]({{< relref "/samples/simple/azure-kubernetes-service/troubleshooting#weblogic-kubernetes-operator-installation-failure" >}}).
+{{% /notice %}}
+
+{{% notice info %}}
+If you have a Docker image built with domain models following [Model in image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}), you can go to [Create WebLogic domain](#create-weblogic-domain) directly.
```

Comment:
[Model in image] -> [Model in Image] (capital I)

Reply:
Done.